source/isaaclab/isaaclab/envs/mdp/events.py

```diff
@@ -35,6 +35,30 @@
     from isaaclab.envs import ManagerBasedEnv
 
 
+def _validate_scale_range(
+    params: tuple[float, float] | None,
+    name: str,
+    *,
+    allow_negative: bool = False,
+    allow_zero: bool = True,
+) -> None:
+    """Sanity–check (low, high) tuples used with `operation=="scale"`.
```

Comment:
Would be good to clarify all the args properly and the purpose of this function in the docstrings.